test/compress/functions.js

```diff
@@ -93,3 +93,29 @@ issue_485_crashing_1530: {
         this, void 0;
     }
 }
+
+issue_1841: {
+    options = {
+        keep_fargs: false,
+        pure_getters: "strict",
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/1843/commits/acfeaf4e70459799314e6246c35a4d05fb70ebd4